app/components/main-content/PrelaunchSignUpForm.tsx

```diff
@@ -79,12 +60,6 @@ const PrelaunchSignUpForm: React.FC<PrelaunchSignUpFormProps> = ({ setFlowState
 
           <h2>Get exclusive early access to try our product</h2>
 
-          <section aria-label="Sign Up with Google or Github">
-            <h3>sign up with google or github</h3>
-            <Button onClick={signUpWithGoogle}>Continue with Google</Button><br/>
-            <Button onClick={signUpWithGitHub}>Continue with Github</Button>
-          </section>
-
           <section aria-label="Sign Up with Email">
             <h3>or sign up with email</h3>
```

Comment:
I think this section and heading component can be removed now that we only have one sign up section.

Reply:
That’s weird, I thought I removed the buttons for Oauth already. Will definitely remove it

Reply:
Wait, I did remove the oauth section, is that what you meant?

Reply:
You removed the section with the OAuth buttons, but we can also remove the `<section>` and `<h3>` HTML elements for the "sign up with email" section. These elements were only needed to separate the OAuth and Email sections from one another - now that there's only one section, we don't need the elements distinguishing them.

Reply:
Ah, I see this was mentioned in a previous comment - https://github.com/Linkta-org/landing-page/pull/38#discussion_r1573476276

I would consider fixing this part of the UI related to this PR. But if this UI fix is not implemented here, we should make a ticket to ensure it gets done @kinskiwu

Reply:
I agree it is better to remove those lines in this PR.

Reply:
@zhiyi15 I looked at Darren's ticket and I think it is safe to revive the h3. Pls keep the section tag.